Cargo.toml

```diff
@@ -35,27 +35,28 @@ maintainer-scripts = "dpkg/info"
 
 [dependencies]
 chrono = "0.4"
-clap = "2.32"
+clap = "2.33"
 env_logger = "0.6"
 failure = "0.1"
-futures = "0.1"
+futures = { version = "0.3", default-features = false, features = ["std", "async-await"] }
 hex = "0.3"
 itertools = "0.8"
 log = "0.4"
 regex = "1"
 serde = { version = "1.0", features = ["derive"] }
 serde_yaml = "0.8"
-tox = "0.0.10"
+tox = { version = "0.1.0", git = "https://github.com/tox-rs/tox.git" }
+tox_crypto = { version = "0.1.0", git = "https://github.com/tox-rs/tox.git" }
```

Comment:
Why don't we reexport crypto from tox crate?

Reply:
Because we forgot to add `tox_crypto` into `tox` crate.